bin/sdk-cli.js

```diff
@@ -10,41 +10,51 @@ const path = require( 'path' );
 const spawnSync = require( 'child_process' ).spawnSync;
 const yargs = require( 'yargs' );
 
-const buildBlock = argv => {
-	const compiler = path.resolve( __dirname, 'create-scripts/block.js' );
-	const editorScript = path.resolve( __dirname, '../', argv.editorScript );
+/**
+ * Internal dependencies
+ */
+const gutenberg = require( './sdk/gutenberg.js' );
 
-	spawnSync( 'node', [ compiler, editorScript, ( argv.outputDir || '' ) ], {
-		env: {
-			SKIP_FLAG_IMAGES: true,
-		},
-		shell: true,
-		stdio: 'inherit',
-	} );
-};
+// Script name is used in help instructions;
+// pick between `npm run calypso-sdk` and `npx calypso-sdk`.
+// Show also how npm scripts require delimiter to pass arguments.
+const calleeScript = path.basename( process.argv[ 1 ] );
+const scriptName = calleeScript === path.basename( __filename ) ? 'npm run calypso-sdk' : calleeScript;
+const delimit = scriptName.substring( 0, 3 ) === 'npm' ? '-- ' : '';
```

Comment:
Not sure if this is overcomplicated at this point but since we don't have any documentation, having clear `--help` instructions is important.